test/spec/SearchResults/getFacetValues.js

```diff
@@ -237,6 +237,36 @@ test('getFacetValues(disjunctive) returns escaped facet values', function() {
   expect(facetValues.length).toBe(3);
 });
 
+test('getFacetValues introduces numeric disjunctive refinements', function() {
+  var searchParams = new SearchParameters({
+    index: 'instant_search',
+    disjunctiveFacets: ['type'],
+    disjunctiveFacetsRefinements: {
+      type: ['5', 50]
+    }
+  });
+
+  var result = {
+    query: '',
+    facets: {
+      type: {}
+    }
+  };
+
+  var results = new SearchResults(searchParams, [result, result]);
+
+  var facetValues = results.getFacetValues('type');
+
+  var expected = [
+    {name: '5', escapedValue: '5', count: 0, isRefined: true},
+    // even though this could be considered refined, it's not because it's a number (existing bug)
+    {name: '50', escapedValue: '50', count: 0, isRefined: false}
```

Comment:
I haven't changed the behaviour of "injected numeric facets"' isRefined, it really should be true, but that was the case before #889 already

other relevant part of the code:

https://github.com/algolia/algoliasearch-helper-js/blob/4bae51bdefd217f3ef9ab9bf02fb653f3b8ae643/src/SearchResults/index.js#L510-L519